source/_integrations/sighthound.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: "Sighthound"
+description: "Detect people with Sighthound Cloud."
+logo: sighthound-logo.png
+ha_category:
+  - Image Processing
+ha_release: 0.103
+---
+
+Adds integration for person detection with [Sighthound Cloud](https://www.sighthound.com/products/cloud). The Sighthound Developer tier (free for non-commercial use) allows 5000 requests per month. If you need more requests per month you will need to sign up for a production account (i.e. Basic or Pro account).
+
+This integration adds an image processing entity where the state of the entity is the number of people detected in an image. For each person detected, an `image_processing.person_detected` event is fired. The event data includes the entity_id of the image processing entity firing the event, and the bounding box around the detected person. **Note** that in order to prevent accidentally using up your requets to Sighthound, by default the component will not automatically scan images, but requires you to call the image_processing.scan service e.g. using an automation triggered by motion. Alternativley, periodic scanning can be enabled by configuring a `scan_interval`.
```

Comment:
```suggestion
This integration adds an image processing entity where the state of the entity is the number of people detected in an image. For each person detected, an `image_processing.person_detected` event is fired. The event data includes the entity_id of the image processing entity firing the event, and the bounding box around the detected person. **Note** that to prevent accidentally using up your requests to Sighthound, by default the component will not automatically scan images, but requires you to call the `image_processing.scan` service by, e.g., using an automation triggered by motion. Alternatively, periodic scanning can be enabled by configuring a `scan_interval`.
```

Besides that, this paragraph contains a lot of text, which makes it hard to read. Try splitting this a bit.

Reply:
done